tests/tests_fabric/strategies/test_fsdp_integration.py

```diff
@@ -119,9 +134,7 @@ def test_setup_module_move_to_device(fabric_module_mock, move_to_device):
     fabric_model = fabric.setup_module(model, move_to_device=move_to_device)
     fabric_module_mock.assert_not_called()
 
-    assert list(param.device for param in model.parameters()) == []
```

Comment:
In 2.0, this returns 1 parameter, which is in CPU/CUDA. This is expected after the call to  https://github.com/pytorch/pytorch/blob/ed1957dc1989417cb978d3070a4e3d20520674b4/torch/distributed/fsdp/fully_sharded_data_parallel.py#L419 introduced in https://github.com/pytorch/pytorch/pull/87086